src/Umbraco.Web.UI.Client/src/packages/data-type/workspace/views/details/data-type-details-workspace-view.element.ts

```diff
@@ -93,7 +93,10 @@ export class UmbDataTypeDetailsWorkspaceViewEditElement extends UmbLitElement im
 	// Notice, we have implemented a property-layout for each states of the property editor ui picker, in this way the validation message gets removed once the choose-button is gone. (As we are missing ability to detect if elements got removed from DOM)[NL]
 	#renderChooseButton() {
 		return html`
-			<umb-property-layout label="Property Editor" description=${this.localize.term('propertyEditorPicker_title')}>
+			<umb-property-layout
+				data-mark="property:editorUiAlias"
```

Comment:
Should it be "property:editor-ui-alias" to match the dash-case format used by others?

Reply:
I think that the EditorUiAlias is seen as a single word, though I am not completely sure

Reply:
It seems like you accidentally removed label and description from this?